### ISSUE TITLE
Nuxt v0.10.7 install KO

### ISSUE THREAD

**Issue by Valleo** [nuxt/nuxt.js#1683]
Hello,

For a few hours, I have got a problem with nuxt 0.10.7.
I am using this version of nuxt since the release, and i never had any problem.
My last stable build was friday (3 days ago), and I didn't modify my own code since then.

When I try to install the lib, npm give me an error

```
npm WARN Error: EPERM: operation not permitted, scandir 'C:\Users\xerj214\Documents\LPC\lpcweb-backend\node_modules\editorconfig\node_modules'
npm WARN  { Error: EPERM: operation not permitted, scandir 'C:\Users\xerj214\Documents\LPC\lpcweb-backend\node_modules\editorconfig\node_modules'
npm WARN   stack: 'Error: EPERM: operation not permitted, scandir \'C:\\Users\\xerj214\\Documents\\LPC\\lpcweb-backend\\node_modules\\editorconfig\\node_modules\'',
npm WARN   errno: -4048,
npm WARN   code: 'EPERM',
npm WARN   syscall: 'scandir',
npm WARN   path: 'C:\\Users\\xerj214\\Documents\\LPC\\lpcweb-backend\\node_modules\\editorconfig\\node_modules' }
npm ERR! path C:\Users\xerj214\Documents\LPC\lpcweb-backend\node_modules\js-beautify\js\bin\css-beautify.js
npm ERR! code ENOENT
npm ERR! errno -4058
npm ERR! syscall chmod
npm ERR! enoent ENOENT: no such file or directory, chmod 'C:\Users\xerj214\Documents\LPC\lpcweb-backend\node_modules\js-beautify\js\bin\css-beautify.js'
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\xerj214\AppData\Roaming\npm-cache\_logs\2017-09-18T15_27_35_142Z-debug.log
```

As you can see, the error happen on windows, but I tried on Linux too, and the result was the same.
My npm version is 5.4.2
I have tried with 4.2.0 too

Even when I clean my folder node_modules and I install only Nuxt, with the command :
`npm install nuxt@0.10.7`

I have the error.

Do you have a solution for this error ?
Is this a Nuxt dependency that causes the trouble ?

Thanks.

**Comment by KonstantinVlasov** [comment]
Hi! You can try to remove package-lock.json, maybe it helps

**Comment by neonerd** [comment]
Hi, this was a problem with js-beautify node module that is now fixed. See https://github.com/beautify-web/js-beautify/issues/1247 for post-mortem

The only strategy to prevent this happening in the future if another dependency starts failing (either because its author will delete it or upload a malfunctioning version) is to use npm shrinkwrap. See https://docs.npmjs.com/cli/shrinkwrap

I'd recommend this to anyone deploying Node.JS to production systems anyway.

**Comment by Valleo** [comment]
As neonerd said (nice name) the problem is because of js-beautify.
They released the version 1.7 monday morning (the GMT morning) and this release is breaking the build of Nuxt 0.10.7

Normaly, I am using a package-lock.json to prevent this kind of issue, and fix the transitive dependencies, but I didn't notice I was on a branch that did not have one.
My mistake.
